lib/ansible/plugins/list.py

```diff
@@ -0,0 +1,237 @@
+# (c) Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+import os
+
+
+from ansible import constants as C
+from ansible.collections.list import list_collections
+from ansible.errors import AnsibleError
+from ansible.module_utils._text import to_native, to_bytes
+from ansible.plugins import loader
+from ansible.utils.display import Display
+from ansible.utils.path import is_subpath
+from ansible.utils.collection_loader._collection_finder import _get_collection_path
+
+display = Display()
+
+# not real plugins
+IGNORE = {
+    # ptype: names
+    'module': 'async_wrapper',
+    'cache': 'base',
```

Comment:
```suggestion
    'module': ('async_wrapper', ),
    'cache': ('base', ),
```

Reply:
`IGNORE[xxx]` is used as something iterable, and I don't think any of the calling code wants a list of letters :)